### PR TITLE
E2E CI Creates a Second Job After Stopping First Job

### PR DESCRIPTION
**Pull request by charles37**

### CODE REVIEW

**Comment by lschuermann** [comment]
This currently fails because this test is engineered to check for currently faulty behavior, and we should merge this once the queueing rewrite of the Switchboard is complete and fixes this behavior.

**Comment by lschuermann** [comment]
@charles37 The new commit you pushed is still based on the old revision, and the merge queue currently doesn't depend on the E2E test. However, I can change that, and then we can try merging this instead of rebasing.

File: .github/workflows/e2e-tests.yml
Comment (lschuermann) [inline .github/workflows/e2e-tests.yml:180]: I'm pretty sure that this request payload needs to be updated (similar to the one above), but we can try merging and seeing whether the queue catches this.

Comment (lschuermann) [reply .github/workflows/e2e-tests.yml:180]: Nevermind, the status checks on the last commit need to pass before GitHub even attempts this merge. If you could rebase these changes and also adjust this payload, this should be good to go.

**Comment by lschuermann** [comment]
@charles37 Can you rebase this test?

**Comment by charles37** [comment]
> @charles37 Can you rebase this test?
ok doing so now.

**Comment by lschuermann** [comment]
@charles37 ping

**Review by lschuermann — APPROVED** [review]

**Pull request merged**
